### PR TITLE
screenshots: use abspath instead of realpath

### PR DESCRIPTION
**Pull request by lasers**
Removing `f-strings` generates the docs, but no screenshots. I looked at `autodoc.py` and it seems to be parsing everything okay. `make html` also generates everything okay here. I'm not sure what else it could be... Two possibilities.
1) Removing `-0` is absolutely important. 
2) Somebody does not like `os.path.realpath()`.

I'm trying `#2`. :pray:

### CODE REVIEW

**Comment by tobes** [comment]
@lasers this makes me so sad.  why are you merging your own commits when you don't even seem to know whats going on?  merging your own commits is evil in almost any circumstance.

i'll go back under my rock

**Comment by lasers** [comment]
If I make something and it works fine under my rock but will break somebody's rock somewhere else, I have to fix it because there are nobody else familiar with `screenshots.py` than you.

0) I waited for you. You didn't review that PR. I wanted you to trust but verify my code.
1) It was my PR that got merged. I am almost certain what went wrong. It's matter of trying stuffs on somebody's rock.
2) I took time to debug `autodoc.py` just in case and it does not look broken so that leaves us to `../doc/path`. Now I need to try things to retain the path.
    * The old path can process screenshots in different places suggesting we should annoyingly remember to go under a rock first before processing. Now, we can be under any rocks getting our rocks off with the randy screenshots.
3) I do not know how long you will be under your rock. It could take days or weeks. I fix this now.

tldr; I was just trying to keep py3status's rock firmly in place so you did not have to come out of yours... Otherwise, py3status's screenshots will be on the rocks. _Damned if you do, damned if you don't._ :moyai:
